invisible_cities/conftest.py

```diff
@@ -840,12 +840,13 @@ def beersheba_config(Th228_hits, PSFDIR, next100_mc_krmap):
                  , threshold     = 5 * units.pes
                  , same_peak     = True
                  , deconv_params = dict( q_cut         = 10
-                                       , drop_dist     = [16.0] * 2
+                                       , drop_dist     = [16.0, 16.0, 4.]
                                        , psf_fname     = PSFDIR
                                        , e_cut         = 12e-3
                                        , n_iterations  = 100
                                        , iteration_tol = 1e-10
                                        , sample_width  = [15.55] * 2
+                                       , cluster_size  = 3
                                        , bin_size      = [ 1.,  1.]
                                        , diffusion     = (1.0, 0.2)
```

Comment:
This means that the tests now run `drop_isolated_clusters` instead of `drop_isolated_sensors`, right? I'm amazed this didn't break the tests. I think we should keep doing what we did before, no? And eventually add a beersheba test with 3d cluster dropping.